sample/build.gradle

```diff
@@ -5,8 +5,7 @@ repositories {
 }
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "28.0.3"
```

Comment:
this is no longer necessary due to the gradle plugin incorporating this by default and using the latest